governance/polls/Proposal - add WBTC-B vault type - January 11, 2021.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Should we add a new WBTC vault type with lower LR? - January 11, 2021
```

Comment:
I would go with something like 'Investigate implementation of a WBTC-B vault-type with a lower LR', also I don't think titles can have '?' due to some weird thing on the portal.

Reply:
I'd update the filename to match this as well.